packages/kbn-check-mappings-update-cli/current_fields.json

```diff
@@ -459,6 +459,11 @@
     "title",
     "type"
   ],
+  "infra-custom-dashboards": [
+    "assetType",
+    "dashboardSavedObjectIdList",
```

Comment:
Nit: what about naming this `dashboardIDs` or `dashboardIdList`? IMO it's preferable to avoid naming "saved object" in the name here.

Reply:
Good point, I like the `dashboardIdList`